### PR TITLE
PMM-4010 Added Go 1.13 to travis

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/node_exporter?pullRequest=19) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/node_exporter?pullRequest=19) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/node_exporter?pullRequest=19) it.</sub>

File: go.mod [hunk 1]
```diff
@@ -21,3 +21,5 @@ require (
 	golang.org/x/sys v0.0.0-20181116152217-5ac8a444bdc5
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 )
+
+go 1.13
```
Comment (AlekSi) [inline go.mod]: We should revert this change while we keep 1.12.

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ sudo: false
 language: go
 
 go:
+  - 1.13.x
```
Comment (AlekSi) [inline .travis.yml]: Please keep it sorted :)

File: Makefile.common [hunk 3]
```diff
@@ -196,10 +196,10 @@ ifdef GO111MODULE
 	mkdir -p $${tmpModule}/staticcheck && \
 	cd "$${tmpModule}"/staticcheck && \
 	GO111MODULE=on $(GO) mod init example.com/staticcheck && \
-	GO111MODULE=on GOOS= GOARCH= $(GO) get -u honnef.co/go/tools/cmd/staticcheck@next && \
+	GO111MODULE=on GOOS= GOARCH= $(GO) get -u honnef.co/go/tools/cmd/staticcheck && \
 	rm -rf $${tmpModule};
 else
-	GOOS= GOARCH= GO111MODULE=off $(GO) get -u honnef.co/go/tools/cmd/staticcheck
+	GOOS= GOARCH= GO111MODULE=off $(GO) get -u honnef.co/go/tools/cmd/staticcheck
```
Comment (AlekSi) [inline Makefile.common]: Trailing whitespace.
